dask_ml/model_selection/_search.py

```diff
@@ -1181,6 +1181,14 @@ def fit(self, X, y=None, groups=None, **fit_params):
             train/test set.
         **fit_params
             Parameters passed to the ``fit`` method of the estimator
+
+        Notes
+        -----
+        This class performs best when each cross validation split fits into
+        RAM because ``fit`` is called on each cross validation split. For
+        example, if using :class:`~sklearn.model_selection.KFold` with k
+        chunks, a :math:`1 - 1/k` fraction of the dataset should fit into RAM.
```

Comment:
What happens if the sklearn impl is wrapped with `dask_ml.wrappers.Incremental`?

Reply:
Doesn't matter. The model doesn't influence the size of the passed NumPy array.

Dask-searchcv always passed NumPy arrays of size `len(X) * (1 - 1/k)` with K-fold cross validation. I think the relevant test is https://github.com/dask/dask-ml/blob/b812fe5db8fcf6f37d805adafe1e7c8f21a18df8/tests/model_selection/dask_searchcv/test_model_selection.py#L151-L155

`cv` could be `sklearn.model_selection.KFold`. If I'm interpreting this test correctly, this asserts that the data is divided in 3 chunks for both Dask arrays and NumPy arrays. In both cases, 2/3rds of the data is used for fitting and 1/3rd for scoring.